hw05.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "hw05"
+author: "Maria Evers"
+date: "8/3/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+
+##Homework 5 part 1:
+```{r}
+
+library(tidyverse)
+library(readr)
+library(knitr)
+library(stringr)
+library(dplyr)
+
+#loading in the data 
+
+dad_mom <- read_delim("dad_mom.txt", delim = "\t")
```

Comment:
You can also use read_tsv() since you have loaded the tidyverse and readr packages.